libs/ui/paginator/src/lib/paginator/paginator.component.ts

```diff
@@ -315,6 +317,18 @@ export class TsPaginatorComponent implements OnChanges, AfterViewInit {
   @Output()
   public readonly recordsPerPageChange = new EventEmitter<number>();
 
+  /**
+   * Emit an event when previous page button clicked
+   */
+  @Output()
```

Comment:
IMO if we're adding these we should add emitters for first and last page buttons too

Reply:
Hmm... okay. Didn't think of that... it might be useful for some consumers.